packages/base/card-api.gts

```diff
@@ -977,9 +976,6 @@ class LinksTo<CardT extends CardDefConstructor> implements Field<CardT> {
         `linksTo field '${this.name}' cannot deserialize a list of resource ids`,
       );
     }
-    if (isLocalResourceID(value.data)) {
-      throw new Error(`lid is not supported in deserialization`);
-    }
```

Comment:
this logic is incorrect. its not that lid's are not supported in deserialization, it's that they are _ignored_. we should not being making type assertions like this as `lid` is a supported member of the JSON API spec.